parts/chapters/subsections/12.3/NETBALAN.fodt

```diff
@@ -4478,10 +4478,10 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
        <text:p text:style-name="_40_Table_20_Contents">NSTEP is a real value that defines the criteria for the network balancing interval, used to define the frequency of the network balancing algorithm. NSTEP may be a negative number, a value of zero, or a positive number, as described below:</text:p>
        <text:list xml:id="list105921202527668" text:continue-list="list105920613217261" text:style-name="_40_NumberedListTable">
         <text:list-item text:start-value="1">
-         <text:p text:style-name="P16992">Negative Value: If NSTEP is any negative value then the time stepping interval for the network balancing is based on the number of Newton iterations used in updating the well targets, as set by the NUPCOL keyword in the RUNSPEC section, or the NUPCOL parameter on the GCONTOL keyword in the SCHEDULE section. If NUPCOL is defined on both keywords, then the GCONTOL value takes precedence. </text:p>
+         <text:p text:style-name="P16992">Negative Value: If NSTEP is any negative value then the time stepping interval for the network balancing is based on the number of Newton iterations used in updating the well targets, as set by the <text:a xlink:href="#__RefHeading___Toc100030_3115110868">NUPCOL</text:a> keyword in the <text:a xlink:href="#__RefHeading___Toc55591_1778172979">RUNSPEC</text:a> section, or the <text:a xlink:href="#__RefHeading___Toc100030_3115110868">NUPCOL</text:a> parameter on the <text:a xlink:href="#__RefHeading___Toc202994_156692946">GCONTOL</text:a> keyword in the <text:a xlink:href="#__RefHeading___Toc43945_784232322">SCHEDULE</text:a> section. If <text:a xlink:href="#__RefHeading___Toc100030_3115110868">NUPCOL</text:a> is defined on both keywords, then the <text:a xlink:href="#__RefHeading___Toc202994_156692946">GCONTOL</text:a> value takes precedence. </text:p>
```

Comment:
I would prefer the link to be removed from "NUPCOL parameter" as this takes you to NUPCOL keyword rather than GCONTOL keyword. But not sure if it is worth the effort.